src/database/client.rs

```diff
@@ -31,9 +34,16 @@ impl PgDbClient {
         pr_number: PullRequestNumber,
         approver: &str,
         priority: Option<u32>,
+        rollup: Option<RollupMode>,
     ) -> anyhow::Result<()> {
         let pr = self.get_or_create_pull_request(repo, pr_number).await?;
-        approve_pull_request(&self.pool, pr.id, approver, priority).await
+        let rollup = rollup
+            .or(pr
+                .rollup
+                .map(|r| RollupMode::from_str(&r).expect("Rollup mode in the db should be valid")))
+            .unwrap_or(RollupMode::Maybe);
+
+        approve_pull_request(&self.pool, pr.id, approver, priority, &rollup.to_string()).await
```

Comment:
And this function should also receive `RollupMode`, not a string. The DB decides how is the value stored in the DB, outside code shouldn't know it.